doc/apps/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5 - mes
 
 B<openssl> B<dgst> 
 [B<-help>]
-[B<-sha|-sha1|-mdc2|-ripemd160|-sha224|-sha256|-sha384|-sha512|-md4|-md5>]
+[B<-sha1|-mdc2|-ripemd160|-sha224|-sha256|-sha384|-sha512|-md4|-md5>]
```

Comment:
I am sorry, I did not get what you meant. Are you saying, as this list is inaccurate, we should not modify. Even-though the list is not accurate, it contains valid digest names except 'sha'.

On a separate note, actually I have not removed 'sha' from the NAME list so that it can be still supported as man sha(1)